docker/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.7'
+services:
+  es:
+    image: "docker.elastic.co/elasticsearch/elasticsearch:7.12.1"
```

Comment:
We might want keep in mind to support and test against opendistro too. Because `7.10` is the last version that ensures compatibility between `Elasticsearch` and `Open Distro for Elasticsearch`